dev_docs/tutorials/building_a_kibana_distributable.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: kibDevTutorialBuildingDistributable
+slug: /kibana-dev-docs/tutorial/building-distributable
+title: Building a Kibana distributable
+summary: Learn how to build a Kibana distributable
+date: 2021-05-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'build', 'distributable']
+---
+
+On the course of that tutorial it will be possible to learn more about how we can create a Kibana distributable 
+as well as handle its different main configurations.
+
+At any given point, you can get CLI help running the following command:
+
+```bash
+yarn build --help
+```
+
+## Prerequisites
+
+For the general aspects of the build we only require you to use one of the following operating systems:
```

Comment:
should we include macOS instructions here if it's supported?  rpm and dpkg can be installed from brew.